src/app/pages/component-category-list/component-category-list.html

```diff
@@ -1,5 +1,6 @@
 <div class="docs-component-category-list">
   <md-card
+      role="link"
```

Comment:
We would have to add an `<a role="link">` around the contents of each card, though. Isn't it a bit more concise to just add the role-link to the md-card itself?